include/boost/uuid/detail/md5.hpp

```diff
@@ -293,6 +294,25 @@ class md5
         (dst)[2] = (unsigned char)((src) >> 16); \
         (dst)[3] = (unsigned char)((src) >> 24);
 
+/* 
+ * The md5 results are read 4 bytes at a time in hash_to_uuid() from
+ * basic_name_generator.hpp, so the endian must be considered here
```

Comment:
The RFC specifies that:
'''
   Set octets zero through 3 of the time_low field to octets zero
      through 3 of the hash.
'''

I think the issue could instead be on the "to string" output and not on the generation.

Reply:
String generation is not the issue, so I am resolving this comment.